crates/red_knot_python_semantic/resources/mdtest/suppressions/type-ignore.md

```diff
@@ -122,3 +122,36 @@ a = (  # type: ignore
     test + 4  # error: [unresolved-reference]
 )
 ```
+
+## File level suppression
+
+```py
+# type: ignore
+
+a = 10 / 0
+b = a / 0
+```
+
+## File level suppression with leading shebang
+
+```py
+#!/usr/bin/env/python
+# type: ignore
+
+a = 10 / 0
+b = a / 0
+```
+
+## Invalid own-line suppression
+
+```py
+"""
+File level suppressions must come before any non-trivia token,
+including module docstrings. 
+"""
+
+# type: ignore
```

Comment:
Should this emit a diagnostic? Or not necessary, because it just won't work and that will be obvious from other diagnostics in the file?

Reply:
This gets flagged by the `unused-ignore-comment` rule


https://github.com/astral-sh/ruff/blob/micha/unused-ignore-comment/crates/red_knot_python_semantic/resources/mdtest/suppressions/type-ignore.md#invalid-own-line-suppression